apps/projects/contracts/Projects.sol

```diff
@@ -26,6 +29,18 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
 * applying bounties in bulk and accepting fulfillment via this contract
 *******************************************************************************/
 interface Bounties {
+
+  function issueAndActivateBounty(
```

Comment:
This can probably be removed from the interface

Reply:
Agreed